README.md

```diff
@@ -2,6 +2,12 @@
 
 A simple React application for managing music tracks, built with Vite and Material-UI.
 
+## Architecture Decision Records (ADRs)
```

Comment:
It is better to store ADRs not in the main "readme" file. Readme should contain only information about the project, possibilities, commands, environments etc. In simple words - any information that should be useful to  developers to run the project and develop. ADRs do not directly impact on the development flow, they are used mostly for deeper understanding and historical tracking. So placing them at the beginning of the main README.md file adds additional noise which is not useful here